docs/signing_release_candidate.md

```diff
@@ -0,0 +1,66 @@
+# Signing a LND release candidate
+
+When a new version of LND is released, binaries for the lnd and lncli programs
+are provided for various platforms and CPU architectures. The hashes of all
+these binaries are written into a file known as the "Manifest". This Manifest is
+then signed by at least five trusted LND contributors (see `[verify-install.sh](https://github.com/lightningnetwork/lnd/blob/master/scripts/verify-install.sh)`
+script for more details), ensuring that users can trust the binaries they 
+download, knowing they haven't been tampered with or backdoored.
```

Comment:
I would be careful with "promises" here. Maybe we should just say "knowing they haven't been modified during the automated build process"?